protostar/starknet/cheatable_execute_entry_point.py

```diff
@@ -1,5 +1,7 @@
+from dataclasses import dataclass
 import logging
-from typing import TYPE_CHECKING, Optional, Tuple, cast, Any
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, List, Optional, Tuple, cast
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Optional, cast
```

nitpick, we have Python 3.9 available